src/Commands/Dashboard/ViewCommand.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+namespace Pantheon\Terminus\Commands\Dashboard;
+
+use Pantheon\Terminus\Commands\TerminusCommand;
+use Pantheon\Terminus\Site\SiteAwareInterface;
+use Pantheon\Terminus\Site\SiteAwareTrait;
+
+class ViewCommand extends TerminusCommand implements SiteAwareInterface
+{
+    use SiteAwareTrait;
+
+    /**
+     * Open the Pantheon site dashboard in a browser
+     *
+     * @command dashboard:view
+     * @aliases dashboard
+     *
+     * @option string $site_env Site & environment to deploy to, in the form `site-name.env`
+     * @option boolean $print Don't try to open the link, just output it
+     *
+     * @usage terminus dashboard my-awesome-site.env-name --print
```

Comment:
Also, a usage for the just-site-name-and-no-env case and the no-site-name-at-all case.